libexec/pyenv-init

```diff
@@ -260,52 +253,40 @@ function print_shell_function() {
   commands=(`pyenv-commands --sh`)
   case "$shell" in
   fish )
-    cat <<EOS
-function pyenv
-  set command \$argv[1]
+    echo 'function pyenv
```

Comment:
Might be worth using
```bash
echo \
'function pyenv
```

the indentation won't break as badly and the code will be easier to read.